contracts/capacitors/HashChainCapacitor.sol

```diff
@@ -14,6 +14,7 @@ import "./BaseCapacitor.sol";
 contract HashChainCapacitor is BaseCapacitor {
     uint256 private _chainLength;
     uint256 private constant _MAX_LEN = 10;
+    uint256 public maxPacketLength;
```

Comment:
immutable?

Reply:
should have an option to update later on if required, so didnt use immutable.